internal/controller/helmchart_controller.go

```diff
@@ -884,6 +888,7 @@ func (r *HelmChartReconciler) reconcileArtifact(ctx context.Context, _ *patch.Se
 	// Record it on the object
 	obj.Status.Artifact = artifact.DeepCopy()
 	obj.Status.ObservedChartName = b.Name
+	obj.Status.ObservedValuesFiles = b.ValuesFiles
```

Comment:
Can you please update the tests to account for this?

Reply:
I'm working on this…

Reply:
@souleb : Can you suggest which tests would be appropriate to extend here? I've added some tests to `helmchart_controller_test.go`, but I've been struggling to find anything which actually touches this code.

Reply:
OK… I think I got it.